runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/EmptyStreamAssertionTest.java

```diff
@@ -54,6 +55,9 @@
   public SparkTestPipelineOptionsForStreaming commonOptions =
       new SparkTestPipelineOptionsForStreaming();
 
+  @Rule
+  public ClearAggregatorsRule clearAggregatorsRule = new ClearAggregatorsRule();
```

Comment:
`PAssertStreaming#runAndAssertContents` asserts aggregators, so this rule is here to make sure they are cleared before running the tests.

I actually added this rule here after seeing some failures stemming from stale aggregator values.